doc/source/tutorials/aggregating_downscaling_consistency.ipynb

```diff
@@ -366,15 +366,15 @@
    "outputs": [],
    "source": [
     "tutorial_df = pyam.IamDataFrame(pd.DataFrame([\n",
-    "    ['World', 'Primary Energy', 'EJ/y', 7, 15],\n",
-    "    ['World', 'Primary Energy|Coal', 'EJ/y', 4, 11],\n",
-    "    ['World', 'Primary Energy|Wind', 'EJ/y', 2, 4],\n",
-    "    ['region_a', 'Primary Energy', 'EJ/y', 4, 8],\n",
-    "    ['region_a', 'Primary Energy|Coal', 'EJ/y', 2, 6],\n",
-    "    ['region_a', 'Primary Energy|Wind', 'EJ/y', 2, 2],\n",
-    "    ['region_b', 'Primary Energy', 'EJ/y', 3, 6],\n",
-    "    ['region_b', 'Primary Energy|Coal', 'EJ/y', 2, 4],\n",
-    "    ['region_b', 'Primary Energy|Wind', 'EJ/y', 0, 2],\n",
+    "    ['World', 'Primary Energy', 'EJ', 7, 15],\n",
```

Comment:
If I follow the conversation correctly, can this be changed back to `EJ/yr`?

Reply:
done throughout all documents, and inverted the pint-test (to test both `EJ` and `EJ/yr`)